src/test_workflow/smoke_test/smoke_tests_spec/2.x/index-management.yml

```diff
@@ -0,0 +1,74 @@
+# yamllint disable
```

Comment:
Can we accommodate the rules for yaml linting instead of disabling it?
https://github.com/opensearch-project/opensearch-build/blob/main/.yamllint.yml
Since entire testing is based heavily on spec files, it is important for them to be in proper format.

Reply:
Let me see if I can fix them.

Reply:
Fixed the yaml lint in the latest commit. Thanks!